Wasted/Pages/Client/FoodPlacePage.xaml.cs

```diff
@@ -31,7 +32,8 @@ public FoodPlacesPage(FoodPlace selectedFoodPlace)
         public void InitializeViews()
         {
             contentScrollView.BindingContext = SelectedFoodPlace;
-            dealsCollectionView.ItemsSource = SelectedFoodPlace.Deals;
+            dealsCollectionView.ItemsSource = DataOrganizer.FilterDeals(SelectedFoodPlace.Deals,
+                delegate(Deal deal) { return !(deal.quantity == 0); });
```

Comment:
deal.quantity != 0 is a little bit more readable

Reply:
Of course, how could I manage to forget about the existence of != operator :D